test/unit/org/apache/cassandra/metrics/CompactionMetricsTest.java

```diff
@@ -0,0 +1,229 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.metrics;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.Sets;
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.apache.cassandra.cql3.CQLTester;
+import org.apache.cassandra.db.ColumnFamilyStore;
+import org.apache.cassandra.db.compaction.ActiveCompactionsTracker;
+import org.apache.cassandra.db.compaction.CompactionController;
+import org.apache.cassandra.db.compaction.CompactionInfo;
+import org.apache.cassandra.db.compaction.CompactionIterator;
+import org.apache.cassandra.db.compaction.CompactionManager;
+import org.apache.cassandra.db.compaction.OperationType;
+import org.apache.cassandra.db.lifecycle.LifecycleTransaction;
+import org.apache.cassandra.io.sstable.ISSTableScanner;
+import org.apache.cassandra.io.sstable.format.SSTableReader;
+import org.apache.cassandra.schema.MockSchema;
+import org.apache.cassandra.utils.FBUtilities;
+
+import static org.junit.Assert.assertNotNull;
+
+public class CompactionMetricsTest extends CQLTester
+{
+    @Test
+    public void testSimpleCompactionMetricsForCompletedTasks() throws Throwable
+    {
+
+        final long initialCompletedTasks = CompactionManager.instance.getCompletedTasks();
+        Assert.assertEquals(0L, CompactionManager.instance.getTotalCompactionsCompleted());
+        Assert.assertEquals(0L, CompactionManager.instance.getTotalBytesCompacted());
+
+        this.createTable("CREATE TABLE %s (pk int, ck int, a int, b int, PRIMARY KEY (pk, ck))");
+        this.getCurrentColumnFamilyStore().disableAutoCompaction();
+        for (int i = 0; i < 10; ++i)
+        {
+            this.execute("INSERT INTO %s (pk, ck, a, b) VALUES (" + i + ", 2, 3, 4)");
+        }
+
+        this.getCurrentColumnFamilyStore().forceBlockingFlush();
+        this.getCurrentColumnFamilyStore().forceMajorCompaction();
+
+        Assert.assertEquals(1L, CompactionManager.instance.getTotalCompactionsCompleted());
```

Comment:
I gently suggest static imports for the assert methods to match existing practice in the codebase.